src/test/java/org/broadinstitute/hellbender/tools/reference/MummerExecutorUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.reference;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.alignment.MummerExecutor;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class MummerExecutorUnitTest extends CommandLineProgramTest {
+
+    private final String COMPARE_REFERENCES_TEST_FILES_DIRECTORY = toolsTestDir + "/reference/CompareReferences/";
+
+    // need intel build for MUMmer
+    @Test(enabled = false)
+    public void testExecuteMummer() throws IOException {
```

Comment:
we are going to need a test that asserts we get a nice user exception in the even that hte correct installation of mummer doesn't exist for the users machine